specification/keyvault/resource-manager/Microsoft.KeyVault/preview/2020-04-01-preview/examples/ManagedHsm_Update.json

```diff
@@ -0,0 +1,77 @@
+{
+  "parameters": {
+    "resourceGroupName": "hsm-group",
+    "name": "hsm1",
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "api-version": "2020-04-01-preview",
+    "parameters": {
+      "tags": {
+        "Dept": "hsm",
+        "Environment": "dogfood",
+        "Slice": "A"
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "properties": {
+          "tenantId": "00000000-0000-0000-0000-000000000000",
+          "initialAdminObjectIds": [
+            "00000000-0000-0000-0000-000000000000"
+          ],
+          "securityDomainId": "00000000-0000-0000-0000-000000000000",
+          "enableSoftDelete": true,
+          "softDeleteRetentionInDays": 90,
+          "enablePurgeProtection": true,
+          "hsmPoolUri": "https://sample-mhsm.managedhsm.azure.net",
+          "provisioningState": "Succeeded",
+          "statusMessage": "ManagedHsm is functional."
+        },
+        "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/hsm-group/providers/Microsoft.KeyVault/managedHSMs/hsm1",
+        "name": "hsm1",
+        "type": "Microsoft.KeyVault/managedHSMs",
+        "location": "westus",
+        "sku": {
+          "family": "B",
+          "name": "Standard_B1"
+        },
+        "tags": {
+          "Dept": "hsm",
+          "Environment": "dogfood",
+          "Slice": "A"
+        }
+      }
+    },
+    "202": {
+      "body": {
+        "properties": {
+          "tenantId": "00000000-0000-0000-0000-000000000000",
+          "initialAdminObjectIds": [
+            "00000000-0000-0000-0000-000000000000"
+          ],
+          "securityDomainId": "00000000-0000-0000-0000-000000000000",
+          "enableSoftDelete": true,
+          "softDeleteRetentionInDays": 90,
+          "enablePurgeProtection": true,
+          "hsmPoolUri": "https://sample-mhsm.managedhsm.azure.net",
+          "provisioningState": "Updating",
```

Comment:
Not a valid state per recent update. Should we use "Succeeded" instead?

Reply:
Keeping this state as I have made Update also as an async operation.
Keeping 'Update' async will be futuristic and at par with all ARM operations we have. If required we can change it in future.